ports/atmel-samd/mpconfigport.mk

```diff
@@ -59,6 +59,8 @@ CIRCUITPY_ZLIB = 0
 
 # Turn off a few more things that don't fit in 192kB
 
+CIRCUITPY_TERMINALIO_VT100 = 1
```

Comment:
Is this enabling or disabling it?

Reply:
That command is enabling the features. I'll go ahead and submit a commit that disables them...